pkg/audit/verifier.go

```diff
@@ -17,8 +17,8 @@ import (
 	"storj.io/storj/internal/memory"
 	"storj.io/storj/pkg/auth/signing"
 	"storj.io/storj/pkg/identity"
-	"storj.io/storj/pkg/overlay"
 	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/statdb"
```

Comment:
I was thinking about the whole change, and I think merging everything into overlay rather than `statdb` will be a much smaller PR and code will end up clearer. Overlay has much clearer and nicer name than `statdb`.

Reply:
Does it really matter if statdb would be merged to overlay or vice versa? There was an idea that the merged thing is called `nodedb`. I plan "statdb" to be renamed to whatever we decide as the last step.

Reply:
On second thought, I decided to follow your advice and start over in smaller steps. This PR became too big and quite messy. I have a new PR #1570 with the first step of just merging the statdb pkg to overlay pkg.